spock-core/src/main/java/org/spockframework/lang/ISpecificationContext.java

```diff
@@ -24,14 +25,21 @@
 
 @Beta
 public interface ISpecificationContext {
+  @Nullable
   SpecInfo getCurrentSpec();
+  @Nullable
```

Comment:
The nullable annotations here, are not correct, are they? As we just discussed, the methods throw exception and do not return null.